test/SatSim/CacheSpec.hs

```diff
@@ -41,19 +43,27 @@ spec =
   describe "CacheSpec" $ do
     prop "round trips a schedule" $
       genScheduleIndex <&> \schedule ->
-        do
+        ifEnabled $ do
```

Comment:
These tests can't run during `nix build` because they need `redis` to be available